setup.cfg

```diff
@@ -47,6 +47,8 @@ install_requires =
     cvxpy==1.3.2
     aiohttp~=3.9.0
     setuptools<=65.6.3
+    pauli_tracker
```

Comment:
One complication: it looks like pip install fails on Apple silicon if you don't have rust installed. This is because pauli_tracker does not provide wheels for Apple silicon and so it must be installed from source. I think it might be good to add a note to the installation instructions in the readme explaining this.

Reply:
I added Jabalizer as an optional install as well as added comments in the Readme.